### DISCUSSION TITLE
How to disable application type when listing applications in drun

### DISCUSSION THREAD

**Discussion by adastx** [davatorium/rofi#1361] [Q&A] [answered]
When I open drun, applications are shown with extra parentheses with details about the program after the name.

For example:
    Firefox (Web Browser)
    Spotify (Music Player)
    Discord (Internet Messenger)

How do I turn this feature off?

**Comment by TonCherAmi** [comment, marked as answer]
See `-drun-display-format` in the [man page](https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown#matching).

**Reply by adastx** [reply]
Thanks I just found it now and was gonna write the answer in here ^^
